README.md

```diff
@@ -87,3 +87,129 @@ An ESX host can be attached and managed indirectly via vSphere API:
+* `search_domain`: Search domain of ESXi server.
+* `dhcp`: true, false
+* `transport`: A resource reference to a transport type declared elsewhere. Eg: `Transport['vcenter']`
+#### Further Documentation
+[VMware Docs](http://pubs.vmware.com/vsphere-55/index.jsp?topic=%2Fcom.vmware.wssdk.apiref.doc%2Fvim.host.DnsConfig.html)
+
+### esx_ntpconfig
+#### Parameters
+* `server`: ['array','of','ntp','servers']
+* `transport`: A resource reference to a transport type declared elsewhere. Eg: `Transport['vcenter']`
+
+### esx_powerpolicy
+#### Parameters
+* `current_policy`: 'static','dynamic','low'
+static = High performance
+dynamic = Balanced
+low = Low Power
+* `transport`: A resource reference to a transport type declared elsewhere. Eg: `Transport['vcenter']`
+
+
+### esx_service
+The service name should be in the form of: `ESXi_hostname:<service name`. Eg `esx.example.com:ntpd`
+#### Parameters
+* `running`: true, false
+* `policy`: 'on','off','automatic'
+* `transport`: A resource reference to a transport type declared elsewhere. Eg: `Transport['vcenter']`
+
+### esx_syslog
+#### Parameters
+* `default_rotate`: The maximum number of log files to keep locally on the ESXi host in the configured logDir. Does not affect remote syslog server retention. Defaults to 8
+* `default_size`: The maximum size, in kilobytes, of each local log file before it is rotated. Does not affect remote syslog server retention. Defaults to 1024 KB.
+* `log_dir`: A location on a local or remote datastore and path where logs are saved to. Has the format `[DatastoreName] DirectoryName/Filename`, which maps to `/vmfs/volumes/DatastoreName/DirectoryName/Filename`. The `[DatastoreName]` is case sensitive and if the specified DirectoryName does not exist, it will be created. If the datastore path field is blank, the logs are only placed in their default location. If `/scratch` is defined, the default is `[]/scratch/log`.
+* `log_host`:A remote server where logs are sent using the syslog protocol. If the logHost field is blank, no logs are forwarded. Include the protocol and port, similar to `tcp://hostname:514`
+* `log_dir_unique`: A boolean option which controls whether a host-specific directory is created within the configured logDir. The directory name is the hostname of the ESXi host. A unique directory is useful if the same shared directory is used by multiple ESXi hosts. Defaults to false.
+* `transport`: A resource reference to a transport type declared elsewhere. Eg: `Transport['vcenter']`
+
+### esx_system_resource
+#### Parameters
+This resource allows the configuration of system resources of a host that are viewed und er the 'System Resource Allocation' section of the vSphere client
+* `host`:
+* `system_resource`:
+* `cpu_limit`:
+* `cpu_reservation`:
+* `cpu_expandable_reservation`:
+* `memory_limit`:
+* `memory_reservation`:
+* `memory_expandable_reservation`:
+* `transport`: A resource reference to a transport type declared elsewhere. Eg: `Transport['vcenter']`
```

Comment:
There is also cpu_unlimited and memory_unlimited. Boolean values that allow enabling an unlimited resource. They are not to be used in conjunction with cpu_limit or memory_limit respectively